README.md

```diff
@@ -1,41 +1,34 @@
-# [WIP] skupper [WIP]
+# Skupper
 
-Note: this repository is a work in progress targeted for the consolidation
-and refactoring of the skupper implementation.
+Skupper enables cloud communication by enabling you to create a Virtual Application Network.
 
-Command line tool for setting up and managing skupper installations
+This application layer network decouples addressing from the underlying network infrastructure.
+This enables communication without a VPN.
```

Comment:
@grs Maybe 'This enable communication in scenarios where you would typically use a VPN'?

Reply:
Let's add 'secure' in there, then I am delighted! :-)